docker/docker-compose.yml

```diff
@@ -144,7 +144,7 @@ services:
     image: quay.io/ukhomeofficedigital/hocs-workflow
     # image: hocs-workflow-test
     ports:
-    - 8081:8080
+    - 8091:8090
```

Comment:
port change incorrect, it should still point to 8080 on docker so please change this to:
- 8091:8080